index.js

```diff
@@ -114,6 +115,11 @@ exports.init = function (Implementation) {
             directorySmartImplementation = path.resolve('.') + '/forest';
           }
 
+          if(directorySmartImplementation){
```

Comment:
`directorySmartImplementation` is always set, right? Then the `if` condition is not necessary I think.